lib/utils/debug-mode.ts

```diff
@@ -0,0 +1,7 @@
+export function isDebugModeEnabled() {
```

Comment:
@pdcastro thanks for the thorough feedback. Yes, I tried to place the function in `utils/helpers` but that broke the build somehow, so I moved it to its own file. I don't understand the build process well enough to debug exactly why. I had to presume it was because of side effects as you mentioned. 

How about `isDebugMode()`?  It's not obvious from the naming of `evalDebug` and `DEBUG` that the value is a `boolean`. `isDebug` is indeed shorter but does saving the 4 characters of `mode` justify the loss of context?

Reply:
@pdcastro @codewithcheese Bare in mind that it was not a user that opened this issue (only me being pedantic).  So I think we should solve it as conveniently as possible.  If we can stick with the global `if (process.env.DEBUG)` then I think that's preferable, since we don't have to change any code, and no need for extra imports into modules.  With that in mind, personally I prefer Paulo's approach of the env var "cleaning" function `evalDebug`, since it only needs to be called once, and doesn't change anything else.  I don't think the name of this function is very important, since it's not used anywhere else.

Reply:
I can see you both share a concern about adding yet another module and changing the usage in so many places. I have updated the PR with an inline assignment of `process.env.DEBUG` at the entrypoints.